cmd/autoheal/healer.go

```diff
@@ -188,6 +182,16 @@ func (h *Healer) Run(stopCh <-chan struct{}) error {
 	go server.ListenAndServe()
 	glog.Info("Web server started")
 
+	// Add a listener that will reload the config files
+	// on config file modification.
+	h.loader.ConfigFilesObserver.AddListener(func(e interface{}) {
```

Comment:
This doesn't feel right: the healer should not know that the loader has a `ConfigFilesObserver` field as that is an implementation detail. The `AddListener` method (or similiar) should be a method of the config, not of the loader, and it should't leak the details of the underlying implementation.